src/output_map/output.rs

```diff
@@ -130,14 +125,18 @@ impl Output {
         physical: PhysicalProperties,
         mode: Mode,
         possible_modes: Vec<Mode>,
-        imgui_context: Context,
-        imgui_pipeline: Renderer,
         active_workspace: String,
         logger: slog::Logger,
     ) -> Self
     where
         N: AsRef<str>,
     {
+        let egui = EguiState::new();
+        let mut visuals = egui::style::Visuals::default();
+        visuals.window_corner_radius = 0.0;
```

Comment:
Minor nitpick, visuals does not have to be mutable in order to construct it with default.
```suggestion
        let visuals = egui::style::Visuals {
               window_corner_radius: 0.0,
               ..Default::default()
        };
```

Reply:
Oh, nice, good idea. Changed :)